src/client/api/object.c

```diff
@@ -162,7 +162,7 @@ daos_obj_fetch(daos_handle_t oh, daos_handle_t th, uint64_t flags,
 	tse_task_t	*task;
 	int		rc;
 
-	rc = dc_obj_fetch_task_create(oh, th, flags, dkey, nr, iods, sgls,
+	rc = dc_obj_fetch_task_create(oh, th, flags, dkey, nr, iods, sgls, NULL,
```

Comment:
Hmm, the extra arg has to be inside the task? can it be into obj_auxi_args? 
This is a general API, so it is probably not good to put extra special arg into the interface.

Reply:
at this time the task is to be created. probably can use tse_task_stack_push() to get the obj_auxi and set the extra arg pointer, and then pop it. that seems more complex, i can change to that way if you prefer it.
At my side, it seems not bad to add void *extra_arg to task args and task create API, and explains by different flags?